src/aggregations/bucket-aggregations/composite-agg-values-sources/values-source-base.js

```diff
@@ -105,6 +105,17 @@ class ValuesSourceBase {
         return this;
     }
 
+    /**
+     * Specifies to include documents without a value for a given source in the
+     * response,or not. Defaults to `false` (not include.
+     *
```

Comment:
Let's add a note here calling out that this was added in [Elasticsearch v6.4](https://github.com/elastic/elasticsearch/pull/29465) similar to this 👇 
https://github.com/sudo-suhas/elastic-builder/blob/39c0f908b46debddd5e9da191205b54a90444547/src/core/sort.js#L154-L155

Also add it to the `src/index.d.ts` docs.

Reply:
Addressed in https://github.com/sudo-suhas/elastic-builder/pull/108/commits/96fc5174f33a364a69dbf914da499b1f57cff1de